server/api/assets.js

```diff
@@ -198,4 +236,34 @@ router.get('/assets/featured/list', async (req, res, next) => {
   }
 });
 
+router.post('/assets/:id/view', async (req, res, next) => {
+  try {
+    const { id } = req.params;
+
+    const assetStats = await sequelize.assetStats.findOrCreate({
+      where: { assetFingerprint: Buffer.from(id, 'hex') },
+    });
+    await assetStats[0].increment('view_count');
+
+    res.sendStatus(200);
+  } catch (e) {
+    next(e);
+  }
+});
```

Comment:
I think should merge with image download with GET method and file download response